hudi-client/src/test/java/org/apache/hudi/table/TestMergeOnReadTable.java

```diff
@@ -1208,6 +1207,83 @@ public void testRollingStatsWithSmallFileHandling() throws Exception {
     }
   }
 
+  @Test
+  public void testHandleUpdateWithMultiplePartitions() throws Exception {
```

Comment:
Can you add descriptive comments at the function level on what this tests and the steps involved ?

Reply:
done